classes/controllers/grid/issues/IssueGridHandler.inc.php

```diff
@@ -576,7 +576,7 @@ public function publishIssue($args, $request)
                     ->getCollector()
                     ->filterByContextIds([$issue->getJournalId()])
                     ->filterByIssueIds([$issue->getId()])
-                    ->filterByStatus([Submission::STATUS_SCHEDULED])
+                    ->filterByStatus([Submission::STATUS_SCHEDULED, Submission::STATUS_PUBLISHED])
```

Comment:
Can you add a comment above this line that explains why the published status is included? Suggested comment:

> // Include published submissions in order to support cases where two
> // versions of the same submission are published in distinct issues. In
> // such cases, the submission will be STATUS_PUBLISHED but the
> // publication will be STATUS_SCHEDULED.

Reply:
check